### ISSUE TITLE
Run Accessibility checker on the Power App

### ISSUE THREAD

**Issue by paul-tsanet** [tsanetgit/MS_Power_App#63]
Request from Microsoft to run accessibility checks on the Power App.  Some links below for information and free tools that can be used

https://www.accessibilitychecker.org/blog/a11y/
https://www.accessibilitychecker.org/

**Comment by arow44** [comment]
[org65ec56f5.crm.dynamics.com-20260526T182305.html](https://github.com/user-attachments/files/28276267/org65ec56f5.crm.dynamics.com-20260526T182305.html)

I cannot use the online accessibility checker since it is a private web page, so I have used Chrome Lighthouse. Please review and confirm if the format is right.

**Comment by paul-tsanet** [comment]
Looks good.

**Comment by arow44** [comment]
Here is a new full accessibility check.

[Assessment_20260608_PowerApps.html](https://github.com/user-attachments/files/28719453/Assessment_20260608_PowerApps.html)
